fiftyone/core/collections.py

```diff
@@ -5497,11 +5499,25 @@ def select_fields(self, field_names=None, _allow_missing=False):
             field_names (None): a field name or iterable of field names to
                 select. May contain ``embedded.field.name`` as well
 
+            meta_filter (None): a filter expression to filter the returned fields
+                based on their metadata. This can be a string that will be matched
+                against anythin in the description or info, or a dict that will be
+                matched against the info. For example, to select only fields that
+                have a description that contains the string "my description", you
+                can use ``meta_filter="my description"``. To select only fields
+                that have a specific key in their info, you can use
+                ``meta_filter=dict(key="value")``.
+
         Returns:
             a :class:`fiftyone.core.view.DatasetView`
         """
+
         return self._add_view_stage(
-            fos.SelectFields(field_names, _allow_missing=_allow_missing)
+            fos.SelectFields(
```

Comment:
We've generally tried to maintain parity between `SelectFields` and `ExcludeFields`. Can you also add the `meta_filter` parameter to `ExcludeFields` so that users can *exclude* fields based on a dynamic criteria if they want?